dojo/finding/views.py

```diff
@@ -2675,7 +2675,7 @@ def finding_bulk_update_all(request, pid=None):
         finding_to_update = request.POST.getlist("finding_to_update")
         finds = Finding.objects.filter(id__in=finding_to_update).order_by("id")
         total_find_count = finds.count()
-        prods = set([find.test.engagement.product for find in finds])
+        prods = {find.test.engagement.product for find in finds}
```

Comment:
Similar to my comment in `dojo/api_v2/views.py`, I think that this form is less clear and that we can do without the associated rule (`C403`).

Reply:
:+1: